### PR TITLE
add some latin american channels

### PR DESCRIPTION
**Pull request by ghost**
all of these worked when i tested them but i'm not sure if they'll work now, especially nickelodeon mexico, so do anything you need to do before merging

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
nickelodeon fluctuates between 576p and 720p i will fix that later

File: streams/ar.m3u [hunk 1]
```diff
@@ -291,6 +291,8 @@ https://538d0bde28ccf.streamlock.net/live-cont.ar/deportv/playlist.m3u8
 http://190.210.221.119/Processor3_9_chunklist_b1300000.m3u8
 #EXTINF:-1 tvg-id="EcoTV.ar",Eco TV (720p)
 https://stream.arcast.com.ar/ecotv/ngrp:ecotv_all/playlist.m3u8
+#EXTINF:-1 tvg-id="ElGourmetPanregional.ar",El Gourmet Panregional (720p)
+http://kstv.us:8080//live/tH300gvHff/8606073792/46594.m3u8
```
Comment (Alstruit) [inline streams/ar.m3u:295]:
```suggestion
```

File: streams/mx.m3u [hunk 2]
```diff
@@ -118,6 +120,10 @@ https://stream.oursnetworktv.com/latin/cnrtelevision/playlist.m3u8
 https://stream.oursnetworktv.com/latin/encoder29/playlist.m3u8
 #EXTINF:-1 tvg-id="CreaLaTV.mx",CreaLaTV (720p) [Not 24/7]
 https://ythls.armelin.one/channel/UCL1iKKBxY_iPu7iH-1wNdwQ.m3u8
+#EXTINF:-1 tvg-id="DePeliculaLatinAmerica.mx",De Película Latin America (480p)
+http://kstv.us:8080//live/tH300gvHff/8606073792/46576.m3u8
+#EXTINF:-1 tvg-id="DePeliculaPlus.mx",De Película Plus (720p)
+http://kstv.us:8080//live/tH300gvHff/8606073792/46577.m3u8
```
Comment (Alstruit) [inline streams/mx.m3u:126]:
```suggestion
```

**Review by Alstruit — DISMISSED** [review]
Xstream server not allowed.

File: streams/mx.m3u (hunk 2 above)
Comment (c4artisan) [reply streams/mx.m3u:126]: oh they're xtream? i never noticed
that's sad

**Comment by c4artisan** [comment]
nickelodeon is down

**Comment by c4artisan** [comment]
the stream is up again but is now the sd feed sadly

File: streams/mx.m3u [hunk 3]
```diff
@@ -199,8 +203,10 @@ https://5e50264bd6766.streamlock.net/mexiquense/videomexiquense/playlist.m3u8
 https://stream8.mexiserver.com:1936/misurtv/misurtv/playlist.m3u8
 #EXTINF:-1 tvg-id="MonteMaria.mx",Monte Maria (1080p)
 https://616eee1852d6c.streamlock.net/live/myStream/playlist.m3u8
-#EXTINF:-1 tvg-id="MVMNoticias.mx",MVM Noticias [Geo-blocked]
+#EXTINF:-1 tvg-id="MVSTV.mx",MVS TV [Geo-blocked]
 https://dish.akamaized.net/Content/HLS_HLS_CLR/Live/channel(mvs)/variant.m3u8
+#EXTINF:-1 tvg-id="Nickelodeon.mx",Nickelodeon (480p)
+https://indythenerd.serv00.net/nickelodeonhd.m3u8
```
Comment (BellezaEmporium) [inline streams/mx.m3u:209]: looped.

& not quite sure serv00 will like that x)

Comment (c4artisan) [reply streams/mx.m3u:209]: the source stream is very unstable
i'll remove it for now until i find a stable one

File: streams/mx.m3u [hunk 4]
```diff
@@ -77,6 +77,8 @@ https://stream.oursnetworktv.com/latin/pueblamx/playlist.m3u8
 https://stream.oursnetworktv.com/latin/encoder22/playlist.m3u8
 #EXTINF:-1 tvg-id="Canal21Rioverde.mx",Canal 21 Rioverde (720p) [Not 24/7]
 https://stream.oursnetworktv.com/latin/encoder91/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal22Nacional.mx",Canal 22 Nacional (720p)
```
Comment (Alstruit) [inline streams/mx.m3u:80]:
```suggestion
#EXTINF:-1 tvg-id="Canal22Nacional.mx",Canal 22 Nacional (720p) [Geo-blocked]
```
403 error. Geo-blocked?

**Comment by c4artisan** [comment]
sorry for splitting up everything into 14 commits
my compulsivity and impatience can really do weird stuff and this is an example
latest commit will be the final one for this request

**Review by Alstruit — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**